### ISSUE TITLE
[Feature] Easily implement jump-to-source-code in Intellij IDEA

### ISSUE THREAD

**Issue by fzyzcjy** [dart-code-checker/dart-code-metrics#951]
Currently, when running, e.g., `dart run dart_code_metrics:metrics check-unnecessary-nullable lib --monorepo`, it outputs warnings but we have to *manually* find out the corresponding code file and line number, which is tedious.

However, a simple change is sufficient: Instead of output like 

```
lib/something.dart:
    ⚠ some messages
      at lib/something.dart:134:3
```

instead output:

```
lib/something.dart:
    ⚠ some messages
      at /absolute/path/to/lib/something.dart:134:3 
```

Notice the last line: `at /absolute/path/to/lib/something.dart:134:3 `. Also notice seems to need a trailing whitespace.

Then, when we run the same command in intellij shell and observe the outputs, the outputs automagically becomes something like:

![image](https://user-images.githubusercontent.com/5236035/182077075-7bd09d39-f71d-47f0-af7a-5366e8c8128d.png)

The blue link can be clicked. When tapping it, Intellij IDEA will automatically jump to that file, and that line, and even that column!

I guess this change is so simple that it may take less than 10 loc :)

**Comment by fzyzcjy** [comment]
👀

**Comment by incendial** [comment]
@fzyzcjy sorry for the delay, looks reasonable, but interesting that the current format works fine in VS Code. Or the problem is only for --monorepo?

**Comment by fzyzcjy** [comment]
Maybe b/c vscode and intellij works differently? `lib/something.dart:134:3` does not work for me in intellij.

**Comment by incendial** [comment]
Okay, I'll test the absolute path in VS Code and if everything is fine will change the path.

**Comment by fzyzcjy** [comment]
Looking forward to the next release!

**Comment by fzyzcjy** [comment]
"waiting for release" - I wonder when will it be released? Given the complexity of using the master branch (need to modify the downloaded package etc) I hope to use the released version.

**Comment by incendial** [comment]
@fzyzcjy sorry for the delay, had some time off the DCM development, hope to release it on Monday

**Comment by fzyzcjy** [comment]
Take your time :)

**Comment by incendial** [comment]
Available in 4.18.0 🚀
